src/pages/home/home.style.scss

```diff
@@ -4,6 +4,18 @@
   margin-right: auto;
   width: 100%;
 }
+
+.buttonStyling {
```

Comment:
I'd recommend improving this class name (e.g. `home-page-nav-button`) and removing the `!important`s.

Here's a stack overflow answer that explains why it's better not to use `!important`: https://stackoverflow.com/questions/3706819/what-are-the-implications-of-using-important-in-css

Reply:
Thanks! I've just pushed these changes